build.gradle

```diff
@@ -148,6 +148,7 @@ dependencies {
     compile group: 'mrj', name: 'MRJ141Stubs', version:'0'
     compile group: 'net.sf.squirrel-sql.thirdparty-non-maven', name: 'skinlf', version:'6.7'
     compile group: 'org.apache.xmlgraphics', name: 'fop', version:'2.1'
+    compile group: 'org.apache.pdfbox', name: 'pdfbox', version:'1.8.9'
```

Comment:
I noticed a problem with the approach I'm using. The PDF renderer is not the only renderer we're using, as the print preview in the GUI uses the AWT renderer. The AWT renderer, unfortunately, is not configured by the fop.xconf file, and for fonts can only use installed system fonts. This means, as of now in this branch, it tries to use a system font called "PCGenSans" which, of course, it canot find, and then falls back to default, which is the Times font included in the FOP library that looks a lot unlike the actual print output.

A solution would be to use "Arial" as the alias instead of "PCGenSans". Arial can reasonably be expected to be installed on a user's system, and while rendering the print preview in Arial does not exactly conform to the actal print output, is looks close, and Arial was what we've been using there until now.

Thoughts?